docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,195 @@
+provides organizations with a method to distribute their data in a scalable manner to thousands of jobs without needing
+to pre-stage data across sites or operate their own scalable infrastructure.
+
+[Stash Caches](/data/stashcache/install-cache) transfer data to clients such as jobs or users. A set of caches are operated across the OSG for the benefit of nearby sites;
+in addition, each site may run its own cache in order to reduce the amount of data transferred over the WAN.
+This document outlines how to run StashCache in a Docker container.
+
+Before Starting
+---------------
+
+Before starting the installation process, consider the following points:
+
+1. Docker should be running in the host you plan to install the cache
+1. User: `user` should be in the `docker` linux group.
+1. A port for XRootD to listen for incoming connections (over HTTP) (default 8000)
+1. A partition on the host to store cached data
+
+
+Running a Container
+-------------------
+
+To run the container, use `docker run` with the following options, replacing the text within angle brackets with your
+own values:
+
+```console
+user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             opensciencegrid/stash-cache:stable \
+             --volume <HOST PARTITION>:/tmp
+```
+
+The `<HOST PORT>` is the port on your server which will accept caching requests and `<HOST PARTITION>` will be the mount where to store
+
+For example, if you've chosen `8212` as your host port, you can verify that it worked with the command:
+
+```console
+user@host $ curl http://localhost:8212/user/dweitzel/public/blast/queries/query1
+```
+
+Which should output:
+
+```
+>Derek's first query!
+MPVSDSGFDNSSKTMKDDTIPTEDYEEITKESEMGDATKITSKIDANVIEKKDTDSENNITIAQDDEKVSWLQRVVEFFE
+```
+
+Readying for Production
+------------------------
+
+Additional configuration is needed to make StashCache production-ready.
+
+1. Add environment variables to set the name of the cache and the directory of the persistent cache.  It is important to remember that the directory set in the environment variable points to the directory **inside** the container.  It is later mapped to a host directory with the `--volume` option.
+
+An example final `docker run` command:
+```console
+user@host $ docker run --rm --publish <HOST PORT>:8000 \
+             --volume <HOST PARTITION>:/cache \
+             --env-file=/opt/xcache/.env \
+             opensciencegrid/stash-cache:stable
+```
+
+Where the environment file on the docker host, `/opt/xcache/.env`, has the following contents:
+```file
+XC_RESOURCENAME=ProductionCache
+XC_ROOTDIR=/cache
+```
+
+It is recommended to use a container orchestration service such as [docker-compose](https://docs.docker.com/compose/) or [kubernetes](https://kubernetes.io/), or start the XCache container with systemd.
+
+An example systemd service file for xcache.  This will require creating the environment file in the directory `/opt/xcache/.env`. 
```

Comment:
```suggestion
An example systemd service file for StashCache. 
This will require creating the environment file in the directory `/opt/xcache/.env`. 
```